dfdaemon/global/global.go

```diff
@@ -51,6 +51,10 @@ type CommandParam struct {
 	// Registry addr and must exist if dfdaemon is used to mirror mode,
 	// format: https://xxx.xx.x:port or http://xxx.xx.x:port.
 	Registry string
+
+	// TrustHosts includes the trusted hosts as keys of the map which dfdaemon forward their
+	// requests directly when dfdaemon is used to http_proxy mode.
+	TrustHosts map[string]string
```

Comment:
Not related to the change here:

It seems incorrect to repeat the same parameter in options and here, if possible, can we stop adding new fields here as I see, we may have to remove this variable sooner or later.

Reply:
@zhouhaibing089 

Thanks, 
I have no idea about where should i add the new field before the refactor of this.
Refactor of removing global file should be another PR.